packages/btc/src/transaction/build.ts

```diff
@@ -307,7 +308,7 @@ export class TxBuilder {
     }
     const insufficientForChange = changeAmount > 0 && changeAmount < this.minUtxoSatoshi;
     if (insufficientForChange) {
-      const shiftedExpectAmount = targetAmount + changeUtxoNeedAmount;
+      const shiftedExpectAmount = collected + changeUtxoNeedAmount;
```

Comment:
The expected result here will be more than the "insufficientBalance" expected result;
```
if (insufficientBalance) {
      throw TxBuildError.withComment(ErrorCodes.INSUFFICIENT_UTXO, `expected: ${targetAmount}, actual: ${collected}`);
    }
```

Here might be some confusion: initially, the funds are insufficient, prompting users to top up. However, upon returning after the top-up, they still find the balance insufficient.

Reply:
Updated the error message when insufficient balance. Now it should warn the user to deposit more satoshi in order to prevent the change UTXO value validation.